packages/rest/src/rest-server.ts

```diff
@@ -132,14 +126,8 @@ export class RestServer extends Context implements Server {
     };
 
     this.bind(RestBindings.HANDLER).toDynamicValue(() => this.httpHandler);
-
-    this.bind(SequenceActions.FIND_ROUTE).toProvider(FindRouteProvider);
     this.bind(SequenceActions.PARSE_PARAMS).to(parseOperationArgs);
```

Comment:
Please bind `PARSE_PARAMS` at application level too  (I think you will need to write a dummy provider for that).